plugins/transport-nio/src/main/java/org/elasticsearch/http/nio/cors/NioCorsHandler.java

```diff
@@ -201,10 +196,6 @@ private boolean validateOrigin() {
             return true;
         }
 
-        if ("null".equals(origin) && config.isNullOriginAllowed()) {
```

Comment:
This seems needed? Or is this handled in a different way?

Reply:
@danielmitterdorfer actually this is always `false` (there's no code assigning anything to that field in the builder) so this must be handled a different way or we have a bug on our hands ... (looking into that)
Looking at the Netty code it's unused there as well ... I guess I could drop this logic there also?

Reply:
It appears unused indeed and I *think* we don't want/need it but maybe @tbrooks8 or @jasontedor can chime in here?

Reply:
I can't find a single usage of `allowNullOrigin()` in git in any revision. I also don't see why we'd want to support `"null"` for the origin really? (plus it seems this was never set to `true` before anyway)